libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -124,7 +124,9 @@ Improvements and New Features
   ``-DLIBCXX_INSTALL_MODULE_DIR=<path>``. The default location is
   ``${PREFIX}/share/libc++/v1``.
 
-- AddressSanitizer annotations have been added to ``std::basic_string``.
+- AddressSanitizer annotations have been added to ``std::basic_string``
+  external buffers (long strings only).
+  Enabled with all allocators by default.
```

Comment:
The release notes should be readable sentences and not bullet points. I would suggest something along the lines of. Feel free to correct this text when my changes make it incorrect.
```suggestion
- AddressSanitizer annotations have been added to ``std::basic_string``.
  These annotations are enabled for all allocators by default.
  It's only enable for long string, string using the small buffer optimization are not annotated.
```